addons/stock/views/stock_picking_views.xml

```diff
@@ -229,8 +230,9 @@
                 <header>
                     <button name="action_confirm" attrs="{'invisible': [('show_mark_as_todo', '=', False)]}" string="Mark as Todo" type="object" class="oe_highlight" groups="base.group_user"/>
                     <button name="action_assign" attrs="{'invisible': [('show_check_availability', '=', False)]}" string="Check Availability" type="object" class="oe_highlight" groups="base.group_user"/>
-                    <button name="button_validate" attrs="{'invisible': ['|', ('state', 'in', ('waiting','confirmed')), ('show_validate', '=', False)]}" string="Validate" type="object" class="oe_highlight" groups="stock.group_stock_user"/>
-                    <button name="button_validate" attrs="{'invisible': ['|', ('state', 'not in', ('waiting', 'confirmed')), ('show_validate', '=', False)]}" string="Validate" type="object" groups="stock.group_stock_user" class="o_btn_validate"/>
+                    <button name="action_set_done_quantities" attrs="{'invisible': [('show_set_all_done', '=', False)]}" string="Set All Done" type="object" class="oe_highlight" groups="stock.group_stock_user"/>
+                    <button name="button_validate" attrs="{'invisible': ['|', '|', ('state', 'in', ('waiting','confirmed')), ('show_validate', '=', False), ('show_set_all_done', '=', True)]}" string="Validate" type="object" class="oe_highlight" groups="stock.group_stock_user"/>
```

Comment:
The `Validate` button should be visible as soon as you have one line with some quantity_done.
Let say you have one tracked move and one non-tracked move. If you complete the tracked one, you will have the `Set all done` visible and `Validate` invisible. 
That means you cannot validate the tracked move and make a backorder.